Let's Skate/Managers/Posts/PostsManager.swift

```diff
@@ -162,4 +172,45 @@ class PostsManager: NewPostService, FeedPostsService, ProfilePostsService, Explo
             completion(.success(()))
         }
     }
+    
+    func likeUnlikePost(post: Post, completion: @escaping(Result<Post,Error>) -> Void) {
```

Comment:
like : add post id to user likes collection 
unlike : remove post from user likes collection
-> return post with modified property